<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RaidCalcClass</class>
 <widget class="QMainWindow" name="RaidCalcClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1341</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RaidCalc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <layout class="QHBoxLayout" name="parametersHorizontalLayout">
      <item>
       <widget class="QGroupBox" name="basicGroup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Basic Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="5" column="1">
          <widget class="QComboBox" name="comboBoxThreads"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxEvent">
           <property name="maxVisibleItems">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelStage">
           <property name="text">
            <string>Story progress:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="editMinSeed">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>00000000</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelGame">
           <property name="text">
            <string>Game:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelEvent">
           <property name="text">
            <string>Event:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labelMinSeed">
           <property name="text">
            <string>Min seed:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="labelMaxSeed">
           <property name="text">
            <string>Max seed:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelStars">
           <property name="text">
            <string>Stars:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBoxStars">
           <property name="currentIndex">
            <number>5</number>
           </property>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QPushButton" name="buttonFindSeeds">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Find Seeds</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labelThreads">
           <property name="text">
            <string>Threads:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBoxStage">
           <property name="currentIndex">
            <number>4</number>
           </property>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3* unlocked</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4* unlocked</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5* unlocked</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6* unlocked</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="editMaxSeed">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>FFFFFFFF</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxGame">
           <item>
            <property name="text">
             <string>Scarlet</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Violet</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelEventGroup">
           <property name="text">
            <string>Event group:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBoxEventGroup">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="pokemonGroup">
        <property name="title">
         <string>Pok√©mon Filters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="2">
          <widget class="QLabel" name="labelHP">
           <property name="text">
            <string>HP:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="QPushButton" name="buttonResetPokemonFilters">
           <property name="text">
            <string>Reset Filters</string>
           </property>
          </widget>
         </item>
         <item row="7" column="4">
          <widget class="QLabel" name="labelSep6">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QSpinBox" name="spinBoxMaxAtk">
           <property name="maximum">
            <number>31</number>
           </property>
           <property name="value">
            <number>31</number>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QLabel" name="labelSep4">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="labelSpe">
           <property name="text">
            <string>Spe:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QSpinBox" name="spinBoxMinSpA">
           <property name="maximum">
            <number>31</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labelAbility">
           <property name="text">
            <string>Ability:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelShiny">
           <property name="text">
            <string>Shiny:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QLabel" name="labelSep3">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="labelSpA">
           <property name="text">
            <string>SpA:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QLabel" name="labelSep5">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labelNature">
           <property name="text">
            <string>Nature:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelSpecies">
           <property name="text">
            <string>Species:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="comboBoxNature"/>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="labelDef">
           <property name="text">
            <string>Def:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QSpinBox" name="spinBoxMinAtk">
           <property name="maximum">
            <number>31</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QSpinBox" name="spinBoxMinDef">
           <property name="maximum">
            <number>31</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QSpinBox" name="spinBoxMaxSpA">
           <property name="maximum">
            <number>31</number>
           </property>
           <property name="value">
            <number>31</number>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QSpinBox" name="spinBoxMaxDef">
           <property name="maximum">
            <number>31</number>
           </property>
           <property name="value">
            <number>31</number>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="labelMaxIV">
           <property name="text">
            <string>Max IV</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBoxShiny">
           <item>
            <property name="text">
             <string>Any</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>No</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="5">
          <widget class="QSpinBox" name="spinBoxMaxSpD">
           <property name="maximum">
            <number>31</number>
           </property>
           <property name="value">
            <number>31</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="labelAtk">
           <property name="text">
            <string>Atk:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxSpecies"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="labelGender">
           <property name="text">
            <string>Gender:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="5">
          <widget class="QSpinBox" name="spinBoxMaxSpe">
           <property name="maximum">
            <number>31</number>
           </property>
           <property name="value">
            <number>31</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="comboBoxAbility"/>
         </item>
         <item row="1" column="5">
          <widget class="QSpinBox" name="spinBoxMaxHP">
           <property name="maximum">
            <number>31</number>
           </property>
           <property name="value">
            <number>31</number>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="labelSpD">
           <property name="text">
            <string>SpD:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QSpinBox" name="spinBoxMinSpe">
           <property name="maximum">
            <number>31</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="9" column="2" colspan="4">
          <widget class="QPushButton" name="buttonMaxIV">
           <property name="text">
            <string>Max IV</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelTeraType">
           <property name="text">
            <string>Tera Type:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="comboBoxGender">
           <item>
            <property name="text">
             <string>Any</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Male</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Female</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Genderless</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="labelMinIV">
           <property name="text">
            <string>Min IV</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="labelSep1">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="spinBoxMinHP">
           <property name="maximum">
            <number>31</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QLabel" name="labelSep2">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBoxTeraType"/>
         </item>
         <item row="6" column="3">
          <widget class="QSpinBox" name="spinBoxMinSpD">
           <property name="maximum">
            <number>31</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <spacer name="spacer4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="itemGroup">
        <property name="title">
         <string>Item Filters</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="labelMinItemsSum">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Min items sum:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxMinItemsSum">
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="checkboxItemFilters">
           <property name="text">
            <string>Activate Filters</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QPushButton" name="buttonEditFilters">
           <property name="text">
            <string>Edit Filters</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelRaidBoost">
           <property name="text">
            <string>Raid boost:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxRaidBoost">
           <property name="maximum">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="spacer3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableSeeds"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1341</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSeedViewer"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExportSeeds"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionSeedViewer">
   <property name="text">
    <string>Seed Viewer</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExportSeeds">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export Seeds</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RaidCalc.qrc"/>
 </resources>
 <connections/>
</ui>
